content/docs/about/use-cases.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Use Cases"
+description = "Kubeflow use cases"
+weight = 1
+aliases = ["/docs/", "/docs//", "/docs/usecases/"]
++++
+
+## General Overview
+
+The end goal of every organization is to have their ML model successfully running in production  and generating value to the business. But what does it take to reach that point? Before a model ends up in production, there are potentially many steps required to build and deploy a ML model: data loading, verification, splitting, processing, feature engineering, model training and verification, hyperparameter tuning, and model serving. In addition, ML models can require more observation than traditional applications, because your data inputs can drift over time. Manually rebuilding models and data sets is time consuming and error prone. To simplify these requirements and challenges, we introduce Kubeflow.
```

Comment:
Tech writer tip: When using an abbreviation for the first time, we should always provide the full term first, even if we think most people will understand the abbreviation. So, in this case, if we don't add the intro paragraph that I suggested earlier, then you should say:

```
... to have their machine learning (ML) model successfully running ...
```